test/disability-benefits/components/StatusPage.unit.spec.jsx

```diff
@@ -101,4 +102,59 @@ describe('<StatusPage>', () => {
     );
     expect(tree.props.children).to.be.null;
   });
+  it('should render notification', () => {
+    const claim = {};
+
+    const tree = SkinDeep.shallowRender(
+      <StatusPage
+          loading
+          message={{ title: 'Test', body: 'Body' }}
+          claim={claim}/>
+    );
+    expect(tree.props.message).not.to.be.null;
```

Comment:
This looks like its testing that the props are set by being passed in, should it be testing that a notification component is rendered with that message (I could be totally off here, not too sure how `tree.props` works)